source/kraken/realtime.cpp

```diff
@@ -80,6 +80,16 @@ static bool is_handleable(const transit_realtime::TripUpdate& trip_update){
     return false;
 }
 
+static bool is_deleted(const transit_realtime::TripUpdate_StopTimeEvent& event) {
+    if (event.HasExtension(kirin::stop_time_event_status) &&
```

Comment:
Minor: You can return the test directly to shorten code.
To be sure: no need to check the case without extension as it's gonna edsapear soon?